ark/phenotyping/cell_cluster_utils.py

```diff
@@ -394,8 +394,8 @@ def train_cell_som(fovs, channels, base_dir, pixel_data_dir, cell_table_path,
                    pixel_cluster_col='pixel_meta_cluster_rename',
                    pc_chan_avg_name='pc_chan_avg.csv',
                    som_weights_name='cell_som_weights.feather',
-                   weighted_cell_channel_name='weighted_cell_channel.feather',
-                   xdim=10, ydim=10, lr_start=0.05, lr_end=0.01, num_passes=1, seed=42):
+                   weighted_cell_channel_name='weighted_cell_channel.csv',
```

Comment:
Why was `weighted_cell_channel_name` changed back to a CSV here? It was addressed in this issue https://github.com/angelolab/ark-analysis/issues/845 to have it be a feather file (for consistency). The code assumes that it is a feather file (ex. lines 507-509, line 115 in this file).

Reply:
Might have been an artifact of resolving a merge conflict, that was a real messy one 😅.